content/en/docs/Getting Started/triggers.md

```diff
@@ -0,0 +1,349 @@
+<!--
+---
+title: "Getting Started with Triggers"
+linkTitle: "Getting Started with Triggers"
+weight: 3
+description: >
+  Create and run your first Tekton Trigger.
+---
+-->
+
+This tutorial shows you how to
+
++   Install Tekton Triggers.
++   Create a Trigger Template.
++   Create a Trigger Bind.
++   Create an Event Listener.
+
+This guide uses a local cluster with [minikube][].
+
+## Before you begin
+
+1.  Complete the two previous Getting Started tutorials: [Tasks][gs-tasks] and
+    [Pipelines][gs-pipelines]. *Do not clean up your resources*, skip the last
+    section.
+
+1.  Install [curl][] if it's not already available on your system.
+
+## Overview
+
+You can use Tekton Triggers to modify the behavior of your CI/CD Pipelines
+depending on external events. The basic implementation you are going to create
+in this guide comprises three main components:
+
+1.  An `EventListener` object that listens to the world waiting for "something"
+    to happen.
+
+1.  When an event happens, the  Event Listener passes the information
+    associated with that event on to a `TriggerBinding` object, which in turn
+    encodes the information into variables that are then passed on to a
+    `TriggerTemplate` object.
+
+1.  The `TriggerTemplate` object uses the variables to instantiate and run a
+    Pipeline.
```

Comment:
```suggestion
1.  A `TriggerTemplate` object configuring the resources, such as PipelineRuns, that will be created when this event occurs.

1.  `TriggerBindings`, which pass event data to the resources created by the `TriggerTemplate`.
```

Reply:
Reworded a bit, to keep the info confined to this example.